pinot-core/src/main/java/org/apache/pinot/core/segment/index/readers/text/LuceneTextIndexReader.java

```diff
@@ -121,7 +131,7 @@ public MutableRoaringBitmap getDocIds(Object value) {
     try {
       Query query = _queryParser.parse(searchQuery);
       _indexSearcher.search(query, docIDCollector);
-      return getPinotDocIds(docIDs);
+      return getPinotDocIdsFromMappingFile(docIDs);
```

Comment:
I prefer the original name

Reply:
ohh yes, I was initially thinking of implementing two solutions. Reverted